### ISSUE TITLE
Establish brand identity

### ISSUE THREAD

**Issue by patcon** [hyphacoop/organizing#77]
## Shared Description

🗣 **Loomio:** https://loomio.hypha.coop/d/XKgh58Za/establish-identity-and-initial-elements-of-co-op-look-and-feel
:date: **Due date:** N/A
:dart: **Success criteria:** We have a brand identity we're all happy with.

## To Do

- [x] Run first design jam session. [notes](https://hackmd.io/@patcon/ryMxS3okB/https%3A%2F%2Fdocs.google.com%2Fdocument%2Fd%2F1Rg1Ze_6Xf9IoMcMm_Qm-UQjzl1WNhju3Nss1Our6W_o%2Fedit?type=book)
- [x] Deploy mood board website -- @YurkoWasHere 
- [x] "Hand-off" from pitch deck process
- [x] Run another design/planning jam session - - @garrying and @dcwalk (@uditvira can also attend)
- [x] @garrying lead dev of first iteration website
     - new repo, snapshot of existing repo (shallow clone?)
     - tachyons framework
- [x] dc+garry website co-work 4-6 Tues
- [x] @garrying  presentation at all hands
- [x] delivery of assets **2020-04-04**:
   - [x] @garrying cleaned up **wordmark** (no small "worker co-op")
   - **stretch** rethink assets on different platforms (e.g., qb, twitter, linkedin)
   - [x] website => #252
   - look and feel section of handbook => https://github.com/hyphacoop/handbook/pull/66
- [x] announce new website + update based on brand
  - [x] twitter
  - [x] update linked in
  - [x] announce elsewhere #197 
- [x] dc: new issues generated for next iteration
  - [x] next website hyphacoop/hypha.coop/issues/6
  - [x] future branding hyphacoop/organizing/issues/254

## Timeline

1. **Feb 7** Develop pitch deck #136 
    - **mid-Jan** First-pass
    - **end-Jan** Feedback period closes
2. **Feb-Mar** Branding exercise phase #77
      - Branding look book (guidelines) 
      - Letterhead #100
3. **mid-Mar** Initial "in progress" website  #80
4. **Apr 5** Final "re-launched" website

**Comment by patcon** [comment]
In the other notes, we said this was @garrying's task? If correct, you wanna own this issue Garry?

**Comment by patcon** [comment]
From https://github.com/hyphacoop/organizing/issues/70#issuecomment-538146143:

We've been asked a few times by our accountant, and Matt@4pig warned we might get it from other professionals and bank relations: **Why are you a non-profit, and does this really fit what you do?**

I'd love to work together on a solid response to this. Thinking maybe this could go in our branding discussion. (though happy to move elsewhere if not here)

**Comment by patcon** [comment]
- #todo checkin with garry
- #todo internal proposal and/or loomio

**Comment by benhylau** [comment]
Saw notes re: in-person meetings Nov/Dec and I'm going to be away. Not a blocker because I don't feel need to be actively participating in design jam. The things I care hope the hypha brand to represent are already expressed in biz plan discussions. Basically, don't wait on me for this :)

**Comment by patcon** [comment]
Had some convos with @nomadic-labs members and hypha about using branding exercise as a low-stakes collab together. Created a new task to coordinate meeting to discuss, and running the exercise (if we decide to). https://github.com/hyphacoop/organizing/issues/137

**Comment by patcon** [comment]
@garrying Can I assign you as owner of this issue, or should it be someone else?

**Comment by garrying** [comment]
@patcon I'm okay with being assigned to this for now with support from others.

**Comment by luluissaa** [comment]
Hey! Nice to meet y’all!! 

I’m on the train to Toronto right now and let’s plan to meet up soon (with folks who are in town). 

We can either do some freeform chatting to get to know each other or dive right into the facilitated conversations and small exercises on shaping a brand identity. Or both :) 

Cheers,
Luisa Ji

Design Strategist 
(613)878-8155
NomadicLabs.ca

**Comment by dcwalk** [comment]
Hi @garrying -- BizDev was thinking we could move forward on this early next week given your availability on Mon and Tues! Putting some holds in calendar and will confirm with you

**Comment by dcwalk** [comment]
Updated issue with outcomes of call. Full notes: https://meetings.hypha.coop/2020-03-30-business-planning-branding.html

**Comment by benhylau** [comment]
Please ping me when this is ready:
>@garrying cleaned up wordmark (no small "worker co-op")

I am also happy to take on the work of updating social media profile pics with new icon, including QB. Just let me know if you would like me to do that.

If you want to provide an updated graphic for the various header images, I can also swap those on our social media accounts.

**Comment by garrying** [comment]
Updated wordmark:
![800x800](https://user-images.githubusercontent.com/227587/78610932-df1b6080-7833-11ea-87dd-eaaac3026d2b.png)

I'm okay with leaving header images as is for now.

**Comment by benhylau** [comment]
Updated logo on GitHub, Twitter, LinkedIn, and QuickBooks (and synced up the invoice styles).

**Comment by dcwalk** [comment]
Announced on twitter, closing this! Will have the look and feel PR land separately (it's close!)
